op-acceptor/runner/runner.go

```diff
@@ -867,14 +866,19 @@ func processMainTestEvent(event TestEvent, result *types.TestResult, testStart,
 	case ActionFail:
 		*testEnd = event.Time
 		result.Status = types.TestStatusFail
+		// If we have an elapsed time from the event, use it as a fallback
+		if event.Elapsed > 0 && result.Duration == 0 {
+			result.Duration = time.Duration(event.Elapsed * float64(time.Second))
```

Comment:
Why plus one second here?

Reply:
I'm not adding a second, I'm simply converting `event.Elapsed` to a time.Duration and then using that as our `result.Duration`.

```
type TestEvent struct {
	...
	Elapsed float64   // Elapsed time in seconds for the specific action
}
```

Reply:
Ahhhhh 🤦 it's `*` not `+` sorry